src/cryptonote_basic/hardfork.cpp

```diff
@@ -449,7 +449,7 @@ bool HardFork::get_voting_info(uint8_t version, uint32_t &window, uint32_t &vote
 		votes += last_versions[n];
 	threshold = (window * heights[current_fork_index].threshold + 99) / 100;
 	//assert((votes >= threshold) == enabled);
-	earliest_height = get_earliest_ideal_height_for_version(version);
+ 	earliest_height = get_height_for_version(version);
```

Comment:
please remove the space as intendtion

Reply:
if `earliest_height` is `uint64_t(-1)` than the return value `enable` must be `false`

Reply:
`enable` is only true if  current_version >= version, for a version that we don't know about this is false by definition as version numbers are assumed to be continuous